site/content/en/docs/tasks/troubleshooting/troubleshooting_provreq.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Troubleshooting Provisioning Request"
+date: 2024-05-20
+weight: 3
+description: >
+  Troubleshooting the status of a Provisioning Request
+---
+
+This doc is about troubleshooting ProvisioningRequests, an API defined by [Cluster Autoscaler](https://github.com/kubernetes/autoscaler/blob/4872bddce2bcc5b4a5f6a3d569111c11b8a2baf4/cluster-autoscaler/provisioningrequest/apis/autoscaling.x-k8s.io/v1beta1/types.go#L41).
+
+See the details about  Provisioning Request[here](https://cloud.google.com/kubernetes-engine/docs/how-to/provisioningrequest).
+
+Provisioning Requests are created in Kueue with [Provisioning Admission Check Controller](/docs/admission-check-controllers/provisioning/), and are treated by Kueue like an [Admission Check](/docs/concepts/admission_check/). It means Provisioning Requests needs to succeed for a Workload to be admitted.
+
+## Before you begin
+
+Before you begin further investigation make sure these fundamental requirements are met:
+- Your cluster's version is at least 1.28.3-gke.1098000 with the Cluster Autoscaler at least 28.122.0
+- Kueue's version is at least 0.5.0
+- You have enabled the `ProvisioningACC` in [the feature gates configuration](/docs/installation/#change-the-feature-gates-configuration)
+- Your node group enables it. It may vary depending on your cloud provider.
```

Comment:
What does mean `Your node group enables it`?

Reply:
Fixed